projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -177,6 +183,20 @@ public static function clear_cron_cache() {
 		wp_cache_delete( 'alloptions', 'options' );
 	}
 
+	/**
+	 * Create a scheduled update status.
+	 *
+	 * @param string           $schedule_id Request ID.
+	 * @param object           $event       The event object.
+	 * @param \WP_REST_Request $request     The request object.
+	 */
+	public static function create_scheduled_update_status( $schedule_id, $event, $request ) {
+		// Only create the status if this is a CREATE request.
+		if ( empty( $request['schedule_id'] ) ) {
+			self::set_scheduled_update_status( $schedule_id, null, null );
+		}
+	}
```

Comment:
I think we don't need this anymore, as we are reading status directly from the logs now, it is not needed.

Reply:
Thanks, fixed in https://github.com/Automattic/jetpack/pull/36835/commits/a2abfe00811adc8e45bbf5df1aa5126b3dcfa079